File: brian/Egg/C++/grader.cpp
#include "Egg.h"

long long _M, _h;

int _count = 0;

bool is_broken(long long k){
	_count++;
	if(_count > 60){
		printf("Wrong Answer: too many queries\n");
		exit(0);
	}
	if(k > _M){
		printf("Wrong Answer: invalid broken query\n");
		exit(0);
	}
	return k > _h;
}

int main() {
	long long hl, ret;
	ret = scanf("%lld %lld", &_M, &_h);
	hl = height_limit(_M);
	if(hl == _h)
		printf("Accept: %d\n", _count);
	else
		printf("Wrong Answer: incorrect height\n");
}

File: contest/brian/Egg/Java/grader.java
import java.util.*;

public class grader{
	static long _M, h, hl;
	static int count;
	private static Scanner scanner = new Scanner(System.in);
	public static long Init(){
		long x;
		x = scanner.nextInt();
		return x;
	}
	public static boolean is_broken(long k){
		count++;
		if(count > 60){
			System.out.println("Wrong Answer: too many queries");
			System.exit(0);
		}
		if(k > _M){
			System.out.println("Wrong Answer: invalid broken query");
			System.exit(0);
		}
		return k > h;
	}
	public static void main(String args[]){
		int i;
		_M = Init();
		h = Init();
		count = 0;
		hl = Egg.height_limit(_M);
		if(hl == h)
			System.out.println("Accepted: " + count);
		else
			System.out.println("Wrong Answer: incorrect height");
	}
}